Allow access to all fbq events via raw query

Jamiewarb opened the pull request:
As it stands, there are elements of the FB Pixel that the wrapper prevents us accessing.

I've created a query method on the analytics.fbq object which will allow us to bypass the wrapper for things it doesn't yet handle.

This prevents us being limited by the wrapper. It is the same approach MatteoGabriele/vue-analytics takes.

An example use is `Vue.analytics.fbq.query('consent', 'revoke');`, as the wrapper currently does not support this.

https://developers.facebook.com/docs/facebook-pixel/events-advanced-use-cases/v3.1

sobolevn commented on package.json:
```diff
@@ -52,7 +52,7 @@
   "main": "dist/vue-analytics-facebook-pixel.js",
   "repository": {
-    "url": "https://github.com/wemake-services/vue-analytics-facebook-pixel.git",
+    "url": "https://github.com/hex-digital/vue-analytics-facebook-pixel.git",
```
Can you please revert this change?

Jamiewarb replied:
Ah! Thank you. This was a mistake of me using my package temporarily in the mean time.

sobolevn commented on src/index.js:
```diff
@@ -62,6 +62,25 @@ const event = (name, data = {}) => {
+const query = (...args) => {
+  if (!_fbqEnabled()) return
+
+  if (config.debug) {
```
Maybe we should refactor `event()` to use `query()` underneath?

Jamiewarb replied:
Up to you! If you think that's a good pattern, then I can certainly make that change

Jamiewarb commented:
@sobolevn I've updated the event and init methods to use query now, and removed that rogue URL. I force pushed over the top so it doesn't sit in this repo's history.

sobolevn commented:
I will release `v2` on the weekends.
